jenkins/build-swift.sh

```diff
@@ -105,6 +105,10 @@ if test -d swift; then
 	cd swift
 fi
 
+echo "Reinstalling six"
+pip uninstall six
+pip install six
+
```

Comment:
why the dance ?
if needed please add a comment

Reply:
Okay I removed the uninstall because it is unnecessary and provided a comment on why I am installing six. A few lines below, './utils/build-script' is called and gets an error saying it cannot find six.

Reply:
You are going to have a few issue wit this:

* There is no guarantee that pip will be present in the machine. i have seen that the 10.15 machines do not have pip unless you specify it in the image.
* Installing python modules in the machine for a build.. is not ideal. You are using a vm, so is not an issue BUT you are also doing it on machines. Something to consider.

Reply:
Thanks, I will add this to my radar and fix in next PR!